scripts/sct_deepseg.py

```diff
@@ -102,62 +116,115 @@ def get_parser():
     return parser
 
 
+def display_list_models():
```

Comment:
:+1: 
Not sure: should we move `display_list_models` and `display_list_tasks` to `deepseg/models.py`?

Reply:
I asked myself the same thing, I was unsure between scripts/sct_deepseg.py, deeseg/models.py and deepseg/core.py. I don't have a strong preference, glad to move it to wherever makes more sense.

Reply:
Moved it to deepseg/models.py :)